draft-ietf-rats-architecture.md

```diff
@@ -316,6 +316,26 @@ Relying Party:
 : A remote server that will securely grant the Attester
 permission to continue operating (i.e., not reboot) for a period of time
 
+## FIDO Biometric Authentication
+
+In the FIDO protocol, it is the device in the user’s hand that authenticates the human user by biometrics like fingerprints or PIN and password.
+The comparison of the input from the user to the reference or expected values is done by the device, often a mobile phone, not the relying party.
+For the relying party to know that the authentication is trustworthy the relying party needs to know that the authenticator part of the device is trustworthy.
+The FIDO protocol employs attestation for this. 
+
+The FIDO protocol supports several attestation protocols and a mechanism by which new ones can be registered and added. Attestation defined by RATS is thus a candidate for use by FIDO. 
+
+Other biometric authentication protocols such as the Chinese IFAA standard and WeChat Pay as well as Google Pay make use of attestation in one form or another.
+
+Attester: 
+
+:The FIDO Authenticator.
```

Comment:
"authenticator" was lower case in 323. Pick one.

Reply:
I do not think hat the Authenticator component alone is the Attester here. It provides strong Authentication to be used in Evidence generated by the device though.